client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -10,8 +10,9 @@ import { memoize } from 'lodash';
 /**
  * Internal dependencies
  */
-import { getSelectedSiteSlug } from 'state/ui/selectors';
 import config from 'config';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import getJetpackScanThreats from 'state/selectors/get-jetpack-scan-threats';
```

Comment:
I think we can leave out Jetpack out of the name :)

Reply:
Perhaps `getSiteScanThreats`?

Reply:
I like that.